bin/ci

```diff
@@ -77,28 +76,42 @@ function update() {
 
 # Utility function that checks if the circle configuration committed to git
 # corresponds to the state of the project so that the workflows and jobs are
-# run according to what has changed as compred to the "master" branch.
+# run according to what has changed as compared to the "master" branch.
 #
 # Usage: check_configuration
 function check_configuration() {
-    update_sources
+    update
     if \
         git ls-tree master --name-only | grep .circleci/config.yml && \
-        git diff --name-only | grep "^.circleci/config.yml$";
+        git diff --name-only | grep "^.circleci/";
```

Comment:
Finally, you only need this test once you have run the `update` function? Then I would remove the first test, _i.e._

```git ls-tree master --name-only | grep .circleci/config.yml &&```

Reply:
True!